File: CRUD-Operations-Angular/ViewModels/UserViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CRUD_Operations_Angular.Web.ViewModels
{
    public class UserViewModel
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public int Age { get; set; }
        public ICollection<UsersProjectsViewModel> Projects { get; set; }
    }
}

File: CRUD-Operations-Angular/ClientApp/app/components/models/userViewModel.ts
import { UsersProjectsViewModel } from "./usersProjectsViewModel";
export class UserViewModel
{
    public id: number;
    public firstName: string;
    public lastName: string;
    public age: number;
    public projects: UsersProjectsViewModel[];

    public constructor(){}

    public static GetUser(firstName: string, lastName: string, age: number): UserViewModel
    {
        let user: UserViewModel = new UserViewModel();
        user.id = 0;
        user.firstName = firstName;
        user.lastName = lastName;
        user.age = age;
        return user; 
    }
    // public constructor(firstName: string, lastName: string, age: number)
    // {
    //     this.Firstname = firstName;
    //     this.LastName = lastName;
    //     this.Age = age;
    // }
}